src/pages/lectures/modern-java/index.mdx

```diff
@@ -124,168 +81,133 @@ _Java_ - це мова програмування загального приз
 
-// allocate memory
-ArrayList list = new ArrayList();
-list.add('Hi!');
+    ```java
+    // C/C++ language allocate memory
+    int *testArray = malloc(5 * sizeof(int));
+    free(testArray);
 
-// cleanup memory by garbage collector
-```
+    // allocate memory
+    ArrayList list = new ArrayList();
+    list.add('Hi!');
 
-Пам'ять керується автоматично.
+    // cleanup memory by garbage collector
+    ```
 
-  </div>
+    Пам'ять керується автоматично.
+  </>
+</DetailsCard>
+
+
+<DetailsCard>
+  <strong>Java - об'єктно-орієнтована мова</strong>
+  <>
+    в Java є:
+
+    - класи
+    - інтерфейси
+    - абстрактні класи
+    - пакети
+    - успадкування
+
+    <JavaOOSnippet />
+  </>
+</DetailsCard>
+
+
+<DetailsCard>
+  <strong>Java - в якійсь-то мірі функціональна мова</strong>
+
+  <ul>
+      <li> Функції вищого порядку за допомогою функціональних інтерфейсів </li>
+      <li> Декілька абстракцій зі світу функціонального програмування, таких як Optional </li>
+  </ul>
+</DetailsCard>
 
-  </li>
-  <li className="details">
-    <input type="checkbox" id="accordion-1-2" />
-    <label className="summary" htmlFor="accordion-1-2">
-      <strong>Java - об'єктно-орієнтована мова</strong>
-    </label>
-  <div className="details-body">
-в Java есть:
-
-- класи
-- інтерфейси
-- абстрактні класи
-- пакети
-- успадкування
-
-<JavaOOSnippet />
-    </div>
-  </li>
-  <li className="details">
-    <input type="checkbox" id="accordion-1-3" />
-    <label className="summary" htmlFor="accordion-1-3">
-      <strong>Java - в якійсь-то мірі функціональна мова</strong>
-    </label>
-  <div className="details-body">
-    <ul>
-        <li> Функції вищого порядку за допомогою функціональних інтерфейсів </li>
-        <li> Декілька абстракцій зі світу функціонального програмування, таких як Optional </li>
-    </ul>
-  </div>
-  </li>
-</ul>
 
 <Level id="whats-new-in-java" number="2" name="What's new in Java 8-11" difficulty="Intermediate" objectives="В Java 8+ було додано багато нових фіч. Вони дуже сильно вплинули на те, як ми пишемо наші програми. Їх знання та застосування на практиці важні для сучасного розробника" />
 
-<ul>
-<li className="details">
-  <input type="checkbox" id="accordion-2-1" hidden />
-  <label className="summary" htmlFor="accordion-2-1">
-    Functional Interfaces
-  </label>
-  <div className="details-body">
-      <JavaFISnippet />
-  </div>
-</li>
-
-<li className="details">
-  <input type="checkbox" id="accordion-2-2" hidden />
-  <label className="summary" htmlFor="accordion-2-2">
-    Stream API
-  </label>
-  <div className="details-body">
-      <JavaStreamSnippet />
-
-Часто використовувані операції стрімів:
-  - Intermediate
+
+<DetailsCard>
+  Functional Interfaces
+  <JavaFISnippet />
+</DetailsCard>
+
+
+<DetailsCard>
+  Stream API
+  <>
+    <JavaStreamSnippet />
+
+    Часто використовувані операції стрімів:
+
+    - Intermediate
     - sorted
     - filter
     - map
-  - Terminal
+    - Terminal
```

Comment:
Should this space be removed?

Reply:
Made it beautful. What's wrong with the straight line? 
![image](https://user-images.githubusercontent.com/49899827/223845912-ef1adadc-92f6-4f9a-9863-f55447ae5b4a.png)